source/_patterns/00-atomer/01-forms/00-avkrysningsboks.mustache

```diff
@@ -2,25 +2,17 @@
   <fieldset class="form-group mb-0">
     <legend class="a-legend">{{{check-legendText}}}</legend>
 {{/ check-legend }}
-    <div class="form-check a-form-checkboxes {{ isDiscreet }} {{ isCenter }} {{ isStacked }} {{styleModifier}}">
-      {{# checkboxes }}
-        {{^ inactive }}
-          <label class="custom-control custom-checkbox">
-            <input type="checkbox" {{# isChecked }}checked="checked"{{/ isChecked }} class="custom-control-input"{{# data-toggle }} data-toggle="{{ data-toggle }}"{{/ data-toggle }}{{# data-target }} data-target="{{ data-target }}"{{/ data-target }}>
-        {{/ inactive }}
-        {{# inactive }}
-          <div class="custom-control custom-checkbox inactive">
-        {{/ inactive }}
-          <span class="custom-control-indicator"></span>
-          <span class="ml-1{{# checkboxes-title-class }} {{ checkboxes-title-class }}{{/ checkboxes-title-class }}">{{{ checkboxes-title }}}</span>
-        {{^ inactive }}
-          </label>
-        {{/ inactive }}
-        {{# inactive }}
-          </div>
-        {{/ inactive }}
-      {{/ checkboxes }}
-    </div>
+
+    {{# checkboxes }}
+
+      <div class="custom-control {{#isInline}}custom-control-inline{{/isInline}} custom-checkbox a-custom-checkbox {{# inactive }} inactive {{/ inactive }} {{# isDiscreet }} a-form-checkboxes--discret {{/ isDiscreet }}">
+          <input type="checkbox" {{# isChecked }}checked="checked"{{/ isChecked }} class="custom-control-input" {{# data-toggle }} data-toggle="{{ data-toggle }}"{{/ data-toggle }}{{# data-target }} data-target="{{ data-target }}"{{/ data-target }} id="{{checkboxID}}" required {{# inactive }} disabled {{/ inactive }}>

```

Comment:
Don't we need option to customize it  like {{# isRequired }} required {{/ isRequired }} ?  This is done in radio button

Reply:
Yes, I have fixed this.